### PR TITLE
Improve the structure of test vectors

### PR DESCRIPTION
**Pull request by jedisct1**
As recommended by the RFC editor:

- Label test vectors sections as "test-vectors"
- Add a link to the JSON version

See https://mailarchive.ietf.org/arch/msg/cfrg/sWa_WkzOW5IPvQ71Hmz21HsBNVY/

### CODE REVIEW

File: draft-irtf-cfrg-aegis-aead.md [hunk 1]
```diff
@@ -1698,9 +1704,11 @@ IANA is also requested to register the following identifiers in the AEAD Algorit
 
 # Test Vectors
 
+The following test vectors are also available in JSON format at [TEST-VECTORS]. In the JSON test vectors, byte string values are JSON strings containing the hex encoding of the byte strings.
```
Comment (samuel-lucas6) [inline draft-irtf-cfrg-aegis-aead.md]: `[TEST-VECTORS]` should be `{{TEST-VECTORS}}` I believe.

I also wonder if the second sentence can be phrased better, but I see that you've taken it from RFC 9605.

**Review by samuel-lucas6 — CHANGES_REQUESTED** [review]

File: draft-irtf-cfrg-aegis-aead.md (hunk 1 above)
Comment (jedisct1) [reply draft-irtf-cfrg-aegis-aead.md]: I did. But it can still be phrased better. What would you suggest?

Comment (jedisct1) [reply draft-irtf-cfrg-aegis-aead.md]: How about "In this format, byte strings are represented as JSON strings containing their hexadecimal encoding." ?

Comment (samuel-lucas6) [reply draft-irtf-cfrg-aegis-aead.md]: > How about "In this format, byte strings are represented as JSON strings containing their hexadecimal encoding." ?

Yeah that's better.

**Pull request merged**